Libraries/plist/Sources/Format/Binary.cpp

```diff
@@ -176,7 +176,7 @@ Create(void *opaque, ABPRecordType type, void *arg1, void *arg2, void *arg3)
                 if (ReadSeek(opaque, offset, SEEK_SET) < 0)
                     return nullptr;
 
-                string.resize(nchars);
+                string.resize(sizeof(char) * nchars);
```

Comment:
Nit: both here and for UTF16, use something like 'size_t stringsize = sizeof(foo) * nchar'